pyaedt/application/Analysis3D.py

```diff
@@ -1277,6 +1277,117 @@ def import_dxf(
         self.oeditor.ImportDXF(vArg1)
         return True
 
+    @pyaedt_function_handler
+    def import_gds_3d(self, gds_file_path, gds_number_list, elevation_list=None, thickness_list=None, import_method=1):
+        """Import a GDSII file.
+
+        Parameters
+        ----------
+        gds_file_path : str
+            Path to the GDS file.
+        gds_number_list : list
+            List of layer number to import.
+        elevation_list : list, optional
+            List of elevation for each GDS layer number imported.
```

Comment:
@amichel0205 I'd rephrase this sentance, it's not very clear.

Reply:
done